### PR TITLE
Added user communication view with api url.

### PR DESCRIPTION
**Pull request by ZachElkins**
Created user communication view to send email from the front end.

Related to the UI pull request of the branch [ET-731-feedback-form](https://github.com/aradnaev/etabot-ui/pull/10).

### CODE REVIEW

File: etabotsite/etabotapp/urls.py [hunk 1]
```diff
@@ -35,6 +36,7 @@
     url(r'^api/parse_projects/', ParseTMSprojects.as_view(), name="estimate_tms"),
     url(r'^api/atlassian_oauth', AtlassianOAuth.as_view(), name='atlassian_oauth'),
     url(r'^api/vote/', VoteView.as_view(), name="vote"),
```
Comment (aradnaev) [inline etabotsite/etabotapp/urls.py:38]: @ZachElkins , could api/vote/ be replaced with api/user_communication/?
If so please do. Also pleas add unit test for the api/user_communication/ in test_views_models.py following existing unit tests as examples

**Comment by aradnaev** [comment]
@ZachElkins , looks good overall. see my other comment about replacing /vote with your new thing.
Also, can you please build and add your corresponding UI into this branch so we can test ui and backend simultaneously?

**Comment by ZachElkins** [comment]
Added testing for
- Unauthenticated user
- Bad JSON data
- API works

**Comment by aradnaev** [comment]
@ZachElkins , it's working overall, thanks. 2 remaining things:
1) the feedback icon didn't show up for me in the UI
2) if it's easy, please send a confirmation email to the user saying something along the lines "thanks for your communication (included below for reference), we will review shortly. <user communication>"

**Pull request merged**